File: cpp_solutions/2544. Alternating Digit Sum.cpp
class Solution {
public:
    int alternateDigitSum(int n) {
        string s = to_string(n);
        int ans = 0;
        for(int x=0; x<s.size();++x){
            if (x%2==0){
                ans+=int(s.at(x))-int('0');
            } else{
                ans-=int(s.at(x)-int('0'));
            }
        }
        return ans;
    }
};

File: python3_solution_set2/2544. Alternating Digit Sum.py
class Solution:
    def alternateDigitSum(self, n: int) -> int:
        s = str(n)
        return sum([int(s[x]) for x in range(len(str(n))) if x%2==0]) - sum([int(s[x]) for x in range(len(str(n))) if x%2!=0])
